mitiq/typing.py

```diff
@@ -90,6 +80,33 @@ class SUPPORTED_PROGRAM_TYPES(EnhancedEnum):
 ]
 
 
+# Supported quantum programs.
+class SUPPORTED_PROGRAM_TYPES(EnhancedEnum):
+    BRAKET = "braket"
+    CIRQ = "cirq"
+    PENNYLANE = "pennylane"
+    PYQUIL = "pyquil"
+    QIBO = "qibo"
+    QISKIT = "qiskit"
+
+    @classmethod
+    def _python_type(cls, identifier: str) -> QPROGRAM:
+        if identifier == "braket":
```

Comment:
Better to use the enum names here, then you will get static checks and things like IDE autocomplete (e.g. if you go and change this to, say, "bracket", it will fail at static checks (or early compilation) instead of runtime.